src/RpcServer/Settings.cs

```diff
@@ -56,6 +60,9 @@ public record RpcServerSettings
             MaxGasInvoke = (long)new BigDecimal(10M, NativeContract.GAS.Decimals).Value,
             MaxFee = (long)new BigDecimal(0.1M, NativeContract.GAS.Decimals).Value,
             TrustedAuthorities = Array.Empty<string>(),
+            EnableCors = true,
+            KeepAliveTimeout = 1,
```

Comment:
Maybe we should be able to disable it with -1

Reply:
Sorry `KeepAliveTimeout` is in seconds never changed default config to reflex.

Reply:
I mean, if possible to disable the keep alive when -1?

Reply:
yes i added https://github.com/neo-project/neo-modules/pull/812/commits/80a0dc87f2459b70a5c09a2903a54a33fe61149e